gulp/.eslintrc

```diff
@@ -6,8 +6,5 @@
   "extends": [
     "habitrpg/server",
     "habitrpg/esnext",
-  ],
```

Comment:
I think this file can be completely removed as it's exactly the same we have in the root folder of the project, so it's unnecessary

Reply:
yep, not a problem.